pom.xml

```diff
@@ -9,7 +9,7 @@
         <version>2.0.8</version>
     </parent>
     <properties>
-        <eiffel-remrem-publish.version>2.0.26</eiffel-remrem-publish.version>
+        <eiffel-remrem-publish.version>2.0.27</eiffel-remrem-publish.version>

```

Comment:
2.0.27 version got released already, change it to 2.0.28 or rebase the changes to community master

Reply:
Fixed.